.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: ci
+env:
+  TERM: xterm # Makes tput work in actions output
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches-ignore: [ staging.tmp, trying.tmp ] # Run CI for all branches except bors tmp branches
+  pull_request: # Run CI for PRs on any branch
+
+jobs:
+  ci-format:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+    # The type of runner that the job will run on
+    runs-on: ${{ matrix.os }}
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: false # LVGL makefile manually installs the submodule
+      - uses: fiam/arm-none-eabi-gcc@v1
+        with:
+          release: '9-2019-q4' # The arm-none-eabi-gcc release to use.
```

Comment:
FWIW, my machine is currently on `10-2020-q4-major` -- since we have some more aggressive warning checks, probably most useful for CI to be on the newest gcc version

Reply:
9-2019-q4 is what the CI ran on for Travis, but yeah I am happy to bump it here as well.

Reply:
Though I will point out that 9-2019-q4 is the newest version that you can currently get through `apt` on ubuntu-20.04 (which is how we recommend ubuntu users install the toolchain)